oxd-server/src/test/java/org/xdi/oxd/server/HealthCheckTest.java

```diff
@@ -18,7 +18,7 @@ public void testHealthCheck(String host, String opHost) {
 
         String resp = Tester.newClient(host).healthCheck();
         assertNotNull(resp);
-        assertEquals("{\"status\":\"running\"}", resp);
+        assertEquals(new String("{\"status\":\"running\"}").trim(), resp);

```

Comment:
just hit heath check manually and put exact string, `assert` must match exact string, all these `new String` and then `trim` are really redundant.